README.md

```diff
@@ -188,16 +188,23 @@ Here is a snippet of a `Dockerfile` to add in tini (be sure to use it in `CMD` o
 
 ```Dockerfile
 # grab tini for signal processing and zombie killing
-ENV TINI_VERSION v0.9.0
-RUN set -x \
-	&& curl -fSL "https://github.com/krallin/tini/releases/download/$TINI_VERSION/tini" -o /usr/local/bin/tini \
-	&& curl -fSL "https://github.com/krallin/tini/releases/download/$TINI_VERSION/tini.asc" -o /usr/local/bin/tini.asc \
+ENV TINI_VERSION v0.18.0
+RUN set -ex \
+	&& curl -fSL --compressed "https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini-static" -o /usr/local/bin/tini \
+	&& curl -fSL --compressed "https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini-static.asc" -o /usr/local/bin/tini.asc \
```

Comment:
In Alpine, `tini` has been available as a package since Alpine 3.4, so I don't think it's very prudent for us to show how to install a static version just for Alpine's sake (since `apk add --no-cache tini` is going to be superior there in almost every way except the ability to get the latest bleeding edge, which is totally sane as an exercise for the reader).

Reply:
Why does this include `--compressed` now?  IMO we should instead just switch to `wget` which is easier to get right out-of-the-box (no `-f` needed, for example):

```bash
wget -O /usr/local/bin/tini "https://...."; \
```